pyof/v0x04/controller2switch/multipart_reply.py

```diff
@@ -288,7 +290,21 @@ def __init__(self, length=None, table_id=None, duration_sec=None,
         self.cookie = cookie
         self.packet_count = packet_count
         self.byte_count = byte_count
-        self.match = match
+        self.match = match or Match()
+        self.instructions = instructions or ListOfInstruction()
```

Comment:
@renanrodrigo I've suggested this statement `instructions or ListOfInstruction()` but just now I've noticed that when we hava a 'list' attribute we are priorizing `instructions or []`. Sorry for leading you to the wrong statement. Please, change it ;)

Reply:
Shame on you!
(I'm doing it :p )